### DISCUSSION TITLE
Token Balance keep accumulating when using OpenRouter Auto

### DISCUSSION THREAD

**Discussion by Kaifeng-Gao** [danny-avila/LibreChat#2349] [Troubleshooting] [answered]
When using OpenRouter Auto model, the user balance will increase instead of decreasing.

Steps to reproduce:

Set up the OpenRouter endpoint with the provided [configuration](https://docs.librechat.ai/install/configuration/custom_config.html)
Perform: use openrouter/auto for generation
Observe the user balance in the dashboard.

**Comment by danny-avila** [comment]
Thanks looking into this

**Comment by danny-avila** [comment, marked as answer]
Fixed by https://github.com/danny-avila/LibreChat/pull/2350
